# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  cb39e84ed597c16a0eb6e74b93bbf6bd53d8e052
Feature: Unity menubar detached from the main window. CSS changes.

diff --git a/mail/themes/linux/mail/mailWindow1.css b/mail/themes/linux/mail/mailWindow1.css
--- a/mail/themes/linux/mail/mailWindow1.css
+++ b/mail/themes/linux/mail/mailWindow1.css
@@ -9,16 +9,21 @@
   ======================================================================= */
 
 @import url("chrome://messenger/skin/messenger.css");
 @import url("chrome://messenger/skin/primaryToolbar.css");
 @import url("chrome://messenger/skin/folderPane.css");
 @import url("chrome://messenger/skin/messageIcons.css");
 @import url("chrome://messenger/skin/shared/mailWindow1.css");
 
+/* Unity menubar detached from the main window. */
+:root[shellshowingmenubar="true"] unified-toolbar > div > .titlebar-buttonbox-container {
+  visibility: visible !important;
+}
+
 #messengerBox {
   background-color: -moz-Dialog;
 }
 
 :root:not([lwt-tree]) #folderTree {
   background-color: -moz-OddTreeRow;
 }
 
